### ISSUE TITLE
hi, How can I use new version KOLBOT for 1.13D ?

### ISSUE THREAD

**Issue by tlwh163** [mf022/D2LoD-files#1]
I set up a 1.13D DIINET myself.  How can I use new version KOLBOT for 1.13D ?
Can you explain how to operate? thanks!

**Comment by tlwh163** [comment]
Also, my 1.13D server can't be K6BOSS, Because once CUBE keys, the game loses its connection immediately. Can you tell me how to solve this problem?

**Comment by mf022** [comment]
Did you see https://d2bot.discourse.group/t/latest-branch-not-work-with-1-13d/179/5 ?

Did you try to change those files (d2bs.dll,d2m.dll, d2bs.exe) in the trunk branch, or to use directly the https://github.com/kolton/d2bot-with-kolbot/tree/patch-113d branch?

**Comment by tlwh163** [comment]
YES, I used patch-113d(d2bs.dll,d2m.dll, d2bs.exe) in the trunk branch, can work,
but  kolbot Commits on Apr 11, 2019, Can't support this,
I really want to use the new features. like limedrop....
So I am looking for your help.

**Comment by mf022** [comment]
> I really want to use the new features. like limedrop....

idk if you can use these features, even on your own private server, ... maybe @noah is the one who can tell you about [d2bot](https://github.com/noah-/d2bs/commits/master) updates and compatibilities between the d2bot-with-kolbot/trunk and 1.13d d2lod. 
you can try to contact him on discord.
